x-pack/plugins/security_solution/public/overview/pages/overview.tsx

```diff
@@ -48,7 +48,7 @@ const OverviewComponent = () => {
   const endpointMetadataIndex = useMemo<string[]>(() => {
     return [ENDPOINT_METADATA_INDEX];
   }, []);
-  const [, { indexExists: metadataIndexExists }] = useFetchIndex(endpointMetadataIndex, true);
+  const [, { indexExists: metadataIndexExists }] = useFetchIndex(endpointMetadataIndex);
```

Comment:
this change breaks the intended functionality of only checking if the index exists

Reply:
actually this is where we stopped using that parameter: https://github.com/elastic/kibana/pull/149360/files#diff-2b9142306df119b8c31e8a66c9b93b2a83f74239b1655919881165ac06398623R127

Reply:
just did some cleanup

Reply:
does that not break the overview page displaying the <LandingPage /> component?